### ISSUE TITLE
Reflection failure with private nested classess in MetroConfigLoader.loadFromServletContext 

### ISSUE THREAD

**Issue by jarkos** [eclipse-ee4j/metro-jax-ws#95]
Setup: Spring Boot + Embeded Tomcat 9

Issue (copied from old repo https://github.com/javaee/metro-jax-ws/issues/1175):
loadFromServletContext(String) wants to access a method through the runtime object class instead of the interface:

```
final Method method = context.getClass().getMethod("getResource", String.class);
This results in errors, when the class is a private nested class, which is true in case of Tomcat:

```
```
Caused by: java.lang.IllegalAccessException: Class com.sun.xml.ws.assembler.MetroConfigLoader$MetroConfigUrlLoader can not access a member of class org.apache.catalina.core.StandardContext$NoPluggabilityServletContext with modifiers "public"
	at java.lang.reflect.Method.invoke(Method.java:490)
	at sun.reflect.Reflection.ensureMemberAccess(Reflection.java:102)
	at com.sun.xml.ws.assembler.MetroConfigLoader$MetroConfigUrlLoader.loadFromServle
```
Possible solution:
I found @torstenwerner fix for that (https://github.com/torstenwerner/jaxws-ri/commit/e3314d314cf23cbceeefca819eea9887d9ea2b72) but I'm not sure if this break something.

**Comment by torstenwerner** [comment]
Hi, it is a pretty harmless fix. Happy holidays! Am 24.12.2019 12:36 schrieb jarkos <notifications@github.com>:Setup: Spring Boot + Embeded Tomcat 9
Issue (copied from old repo javaee/metro-jax-ws#1175):
loadFromServletContext(String) wants to access a method through the runtime object class instead of the interface:
final Method method = context.getClass().getMethod("getResource", String.class);
This results in errors, when the class is a private nested class, which is true in case of Tomcat:


Caused by: java.lang.IllegalAccessException: Class com.sun.xml.ws.assembler.MetroConfigLoader$MetroConfigUrlLoader can not access a member of class org.apache.catalina.core.StandardContext$NoPluggabilityServletContext with modifiers "public"
	at java.lang.reflect.Method.invoke(Method.java:490)
	at sun.reflect.Reflection.ensureMemberAccess(Reflection.java:102)
	at com.sun.xml.ws.assembler.MetroConfigLoader$MetroConfigUrlLoader.loadFromServle

Possible solution:
I found @torstenwerner fix for that (torstenwerner/jaxws-ri@e3314d3) but I'm not sure if this break something.

**Comment by lukasj** [comment]
fixed by #96
